carball/json_parser/game.py

```diff
@@ -411,6 +411,18 @@ def parse_replay(self):
                         self.ball_type = 'Basketball'
                         data_dict = BallActor.get_data_dict(actor_data, version=self.replay_version)
                         player_ball_data['ball'][frame_number] = data_dict
+                    elif actor_data["TypeName"] == "Archetypes.Ball.Ball_Puck":
+                        if actor_data.get('TAGame.RBActor_TA:bIgnoreSyncing', False):
+                            continue
+                        self.ball_type = 'Puck'
```

Comment:
I feel instead of typing this as a string, we can do a python enum 
```python
class BallType(Enum)
    default = auto()
    basketball = auto()
    puck = auto()
    cubeball = auto()
```
then do `self.ball_type = BallType.puck`

Reply:
Yeah that's much cleaner